BoxOffice/Presentation/View/BoxOfficeCell.swift

```diff
@@ -0,0 +1,155 @@
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.alignment = .leading
+        stackView.distribution = .fillEqually
+        stackView.spacing = 0
+        return stackView
+    }()
+    
+    private let separatorView = UIView()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension BoxOfficeCell {
+    private func configureLayout() {
+        configureLabels()
+        configureStackViews()
+        configureHorizontalStackView()
+        configureSeparatorView()
+    }
+    
+    private func configureLabels() {
+        [rankLabel, rankIntensityLabel, movieNameLabel, audienceAccountLabel].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            $0.adjustsFontForContentSizeCategory = true
+        }
+    }
+    
+    private func configureStackViews() {
+        [leftStackView, rightStackView, horizontalStackView].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            $0.spacing = 0
+        }
+        
+        leftStackView.addArrangedSubview(rankLabel)
+        leftStackView.addArrangedSubview(rankIntensityLabel)
+        rightStackView.addArrangedSubview(movieNameLabel)
+        rightStackView.addArrangedSubview(audienceAccountLabel)
+        horizontalStackView.addArrangedSubview(leftStackView)
+        horizontalStackView.addArrangedSubview(rightStackView)
+        addSubview(horizontalStackView)
+    }
+    
+    private func configureHorizontalStackView() {
+        
+        NSLayoutConstraint.activate([
+            horizontalStackView.centerXAnchor.constraint(equalTo: centerXAnchor),
+            horizontalStackView.centerYAnchor.constraint(equalTo: centerYAnchor),
+            horizontalStackView.widthAnchor.constraint(equalTo: widthAnchor),
+            horizontalStackView.heightAnchor.constraint(equalTo: heightAnchor, multiplier: 0.7)
+        ])
+        
+        let leftStackWidthConstraint = leftStackView.widthAnchor.constraint(equalTo: widthAnchor, multiplier: 0.2)
+        leftStackWidthConstraint.priority = .defaultHigh
+        leftStackWidthConstraint.isActive = true
+    }
+    
+    private func configureSeparatorView() {
+        addSubview(separatorView)
+        
+        separatorView.backgroundColor = .opaqueSeparator
+        separatorView.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate([
+            separatorView.heightAnchor.constraint(equalToConstant: 0.3),
+            separatorView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            separatorView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            separatorView.topAnchor.constraint(equalTo: topAnchor),
+        ])
+    }
+    
+    func matchRankIntensity(of isNew: Bool, with rankNumber: Int) {
```

Comment:
overloading과 overriding의 차이점에 대해서 알려주세요

해당 부분은 반드시 overloading을 사용했을 때 장점이 있는 부분이였나요?

Reply:
해당 함수명은 오버로딩을 의도했다기 보다는, 어울리는 함수명을 작성하다보니 똑같은 함수명으로 오버로딩이 되었습니다. 딱히 이점이 없어서 다른 이름으로 변경하겠습니다. 

1. overriding
A subclass can provide its own custom implementation of an instance method, type method, instance property, type property, or subscript that it would otherwise inherit from a superclass. This is known as overriding.

2. overloading
함수의 파라미터가 다르면 같은 이름으로 함수를 정의해서 사용할 수 있는 것

참고자료: 
1. https://docs.swift.org/swift-book/documentation/the-swift-programming-language/inheritance#Overriding
2. https://babbab2.tistory.com/129